targets/psa_switch/pswitch_CLI.py

```diff
@@ -37,41 +50,69 @@ def __init__(self, pre_type, standard_client, mc_client, pswitch_client):
                                         standard_client, mc_client)
         self.pswitch_client = pswitch_client
 
+    @handle_bad_input
     def do_set_queue_depth(self, line):
         "Set depth of one / all egress queue(s): set_queue_depth <nb_pkts> [<egress_port>]"
         args = line.split()
-        depth = int(args[0])
+        depth = self.parse_int(args[0])
```

Comment:
`self.parse_int(args[0], "nb_pkts")`

Reply:
thanks for catching - fixing this in next commit